tests/test_skewing.py

```diff
@@ -9,15 +12,18 @@ class TestCodeGenSkewing(object):
     '''
     Test code generation with blocking+skewing, tests adapted from test_operator.py
     '''
-    @pytest.mark.parametrize('expr, expected', [
+    @pytest.mark.parametrize('expr, expected, norm_u, norm_v', [
```

Comment:
SO i know these are the norm we get, and that these are quite exotic operators, but are we sure this is the norm we are supposed to get? Like can we make up a simple case were we know by hand this is the norm?

Reply:
The operator is not exotic at all...is just: 'Eq(u.forward, u + 1)'
nothing runs differently. So in the end of this test we also check against the non-skewed version. The only chance this is wrong is to doubt that a devito operator for this:
` Eq(u.forward, u + 1)` is wrong

Any chance you thought that the `expected` args are used as input?

Reply:
`Eq(u.forward, u + 1)` this is trivial to get an explicit expression rather than a number for. i.e if `time_M` -1 you should have `norm_u = sqrt(3*3*3)` which is more explicit and therefore safer/clearer than just a number.

Reply:
I see, will push now

Reply:
@mloubout better now?

Reply:
Looking good